### DISCUSSION TITLE
run error on sd-forge

### DISCUSSION THREAD

**Discussion by rayqianhua** [AIrjen/OneButtonPrompt#218] [Q&A]
it shows error on sd-forge, 
![image](https://github.com/user-attachments/assets/c2797b4e-1711-42fb-8bca-ac782ef7e439)

**Comment by ScionoicS** [comment]
I think is is mainly due to the gradio update. There's a branch to work on code for it, but we're not there yet. Maybe it's a simple fix for forge? i just don't know.

**Comment by AIrjen** [comment]
We have for it in issue: #214 
I need to rework the gradio component for this to work. I have been away for a couple of weeks, so progress has been slow so far!
